packages/components/src/Card/Card.tsx

```diff
@@ -4,23 +4,49 @@ import { XOR } from "ts-xor";
 import styles from "./Card.css";
 import colors from "./colors.css";
 import { CardClickable } from "./CardClickable";
+import { Button, ButtonProps } from "../Button";
 import { Heading } from "../Heading";
 
-interface CardProps {
+export type HeaderProps = XOR<HeaderActionProps, HeaderCustomProps>;
+interface CardCommonProps {
   /**
    * The `accent`, if provided, will effect the color accent at the top of
    * the card.
    */
   readonly accent?: keyof typeof colors;
   readonly children: ReactNode | ReactNode[];
+}
+
+interface CardHeaderProps {
+  readonly title?: never;
```

Comment:
I believe it is not necessary to add `title:Never` as the XOR utility type handles that for us. Personally it made it more confusing for me that title was defined here.